podoglyph/gradebook
# Development

#### What does this PR do?
This PR finishes the teacher_course_show view with testing. It creates the Enrollments table, SemesterCourse table and associated models and specs.

#### How should this be manually tested?
As a Teacher, you can view your course roster and then click on course to see the enrolled students and their grades.

#### Any background context you want to provide?

#### Questions:

  - Do Migrations Need to be ran?
* Yes
  - Do Environment Variables need to be set?
* No
  - Any other deploy steps?
After migrating, run `db:seed` to update the database with student enrollments.
